README.md

```diff
@@ -1,24 +1,174 @@
-# README
+# 📖 [Budget_App] <a name="about-project"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> This is an educational project is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what. The Ruby on Rails application that allows the user to:
+- [x] register and log in, so that the data is private to them.
+- [x] introduce new transactions associated with a category.
+- [x] see the money spent on each category.
 
-Things you may want to cover:
+## Project Documentation 📄
 
-* Ruby version
+- [x] Here is the presentation video [link](https://www.loom.com/share/ca9b387a0a3e4f2e938d3509cfd2c9d6)👈
+<!-- - [x] Here is the deployed app in [Heroku](https://app-budget-on-rails.herokuapp.com/)👈 -->
```

Comment:
- [ ] As pointed out by the previous code reviewer, which is a [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md) for this project, " `The project should be deployed and accessible online.` ",  please deploy your app using Heroku, Netlify or GitHub Pages,
your project should be deployed properly, the home and detail page works as expected, A link to the deployed project should be present in the README.

Reply:
@Mihndim2020 Thank you for the review. Yes I will deploy the app on Heroku as you suggested.